apps/cms/test/paragraph_test.exs

```diff
@@ -273,6 +274,20 @@ defmodule CMS.ParagraphTest do
       assert %TripPlanWidget{} = from_api(api_data)
     end
 
+    test "parses a code embed paragraph" do
+      code_embed_data = api_paragraph("code_embed")
+
+      assert %CodeEmbed{
+               header: header,
+               body: code
+             } = from_api(code_embed_data)
+
+      assert %ColumnMultiHeader{} = header
```

Comment:
If you want to keep the header, could you please test it's parsing behavior? Given that it works differently than the body element I can see that as a potential source of confusion, so calling out the difference seems important.

Reply:
Done. The header component is still scrubbed just like all other paragraph type header components -- this is not the EmbedCode property that deals with raw code embedding.